contracts/crowdsale/validation/IndividuallyCappedCrowdsale.sol

```diff
@@ -20,33 +20,16 @@ contract IndividuallyCappedCrowdsale is Ownable, Crowdsale {
    * @param _beneficiary Address to be capped
    * @param _cap Wei limit for individual contribution
    */
-  function setUserCap(address _beneficiary, uint256 _cap) external onlyOwner {
+  function setCap(address _beneficiary, uint256 _cap) external onlyOwner {
     caps_[_beneficiary] = _cap;
   }
 
-  /**
-   * @dev Sets a group of users' maximum contribution.
-   * @param _beneficiaries List of addresses to be capped
-   * @param _cap Wei limit for individual contribution
-   */
-  function setGroupCap(
-    address[] _beneficiaries,
-    uint256 _cap
-  )
-    external
-    onlyOwner
-  {
-    for (uint256 i = 0; i < _beneficiaries.length; i++) {
-      caps_[_beneficiaries[i]] = _cap;
-    }
-  }
-
   /**
    * @dev Returns the cap of a specific user.
```

Comment:
This and other comments reference 'users': we should probably rename those to 'accounts' or 'beneficiaries'